chain/chain/src/runtime/tests.rs

```diff
@@ -1771,3 +1765,60 @@ fn test_prepare_transactions_empty_storage_proof() {
 
     assert!(validation_result.is_err());
 }
+
+#[test]
+#[cfg_attr(not(feature = "test_features"), ignore)]
+fn test_storage_proof_garbage() {
+    if !checked_feature!("stable", StatelessValidationV0, PROTOCOL_VERSION) {
+        return;
+    }
+    let shard_id = 0;
+    let signer = create_test_signer("test1");
+    let env = TestEnv::new(vec![vec![signer.validator_id().clone()]], 100, false);
+    let garbage_size_mb = 50usize;
+    let receipt = Receipt::V1(ReceiptV1 {
+        predecessor_id: signer.validator_id().clone(),
+        receiver_id: signer.validator_id().clone(),
+        receipt_id: CryptoHash::hash_bytes(&[42]),
+        receipt: near_primitives::receipt::ReceiptEnum::Action(ActionReceipt {
+            signer_id: signer.validator_id().clone(),
+            signer_public_key: signer.public_key(),
+            gas_price: env.runtime.genesis_config.min_gas_price,
+            output_data_receivers: vec![],
+            input_data_ids: vec![],
+            actions: vec![Action::FunctionCall(
+                FunctionCallAction {
+                    method_name: format!("internal_record_storage_garbage_{garbage_size_mb}"),
```

Comment:
why add the size in the function name instead of adding it as an argument?

Reply:
Added to the PR description:
> Having size as part of the method name (and not as an argument) makes it easier to send a transaction using near cli (`args` needs to be base64 encoded).